openapi.yaml

```diff
@@ -14656,6 +15387,168 @@ components:
             The number of GPUs this Linode Type offers.
           example: 0
           x-linode-cli-display: 11
+    LKEClusterRequestBody:
+      type: object
+      description: A Kubernetes cluster.
+      properties:
+        label:
+          type: string
+          description: >
+            This Kubernetes cluster's unique label for display purposes only.
+            If no label is provided, one will be assigned automatically.
+
+            Labels have the following constraints:
+              * Must start with an alpha character
+              * Must only consist of alphanumeric characters and dashes (`-`)
+              * Must not contain two dashes in a row
+          minLength: 3
+          maxLength: 32
+          # Kubernetes does NOT allow underscores in resource names and we are
+          # passing this name to Kubernetes. Thus the following pattern:
+          pattern: '[a-zA-Z0-9-]{3, 32}'
+          example: lkecluster12345
+        region:
+          type: string
+          description: This Kubernetes cluster's location.
+          example: us-central
+        version:
+          type: string
+          description: >
+            The desired Kubernetes version for this Kubernetes cluster.
```

Comment:
could add, "In the format of <major>.<minor>, the latest supported patch version will be deployed"